docs/workflows/verify-images.rst

```diff
@@ -0,0 +1,262 @@
+            "download_concurrency": null,
+            "exclude_tags": null,
+            "headers": null,
+            "include_tags": [
+                "8.5"
+            ],
+            "max_retries": null,
+            "name": "legacy-registry-rh",
+            "policy": "immediate",
+            "proxy_url": null,
+            "pulp_created": "2022-03-10T15:06:02.107728Z",
+            "pulp_href": "/pulp/api/v3/remotes/container/container/64da87b6-7bff-4ab5-9b35-7922e3efd352/",
+            "pulp_labels": {},
+            "pulp_last_updated": "2022-03-10T15:06:02.107769Z",
+            "rate_limit": null,
+            "sigstore": "https://access.redhat.com/webassets/docker/content/sigstore",
+            "sock_connect_timeout": null,
+            "sock_read_timeout": null,
+            "tls_validation": true,
+            "total_timeout": null,
+            "upstream_name": "ubi8/ubi-micro",
+            "url": "https://registry.access.redhat.com"
+        }
+
+After sync task completion, create distribution with the base_path ``local-ubi8-repo`` under which
+the image and signatures will be available for pull and verification:
+
+::
+
+        $ http https://fluffy.example.com/pulp/api/v3/distributions/container/container/1e59e5a8-5247-401d-a9b5-9b3ef07d5efe/
+        HTTP/1.1 200 OK
+        Access-Control-Expose-Headers: Correlation-ID
+        Allow: GET, PUT, PATCH, DELETE, HEAD, OPTIONS
+        Connection: keep-alive
+        Content-Length: 659
+        Content-Type: application/json
+        Correlation-ID: ab2d64376e2849c5bc0cd1f999355115
+        Date: Thu, 10 Mar 2022 16:08:25 GMT
+        Referrer-Policy: same-origin
+        Server: nginx/1.20.1
+        Strict-Transport-Security: max-age=15768000
+        Vary: Accept, Cookie
+        X-Content-Type-Options: nosniff
+        X-Frame-Options: DENY
+
+        {
+            "base_path": "local-ubi8-repo",
+            "content_guard": "/pulp/api/v3/contentguards/core/content_redirect/27d55db4-2b99-49f6-9838-8ca52647d714/",
+            "description": null,
+            "name": "local-ubi8-repo",
+            "namespace": "/pulp/api/v3/pulp_container/namespaces/fe4d8115-a81c-4eb3-950e-2cf1cc7f033f/",
+            "private": false,
+            "pulp_created": "2022-03-10T16:04:52.026832Z",
+            "pulp_href": "/pulp/api/v3/distributions/container/container/1e59e5a8-5247-401d-a9b5-9b3ef07d5efe/",
+            "pulp_labels": {},
+            "registry_path": "fluffy.example.com/local-ubi8-repo",
+            "repository": null,
+            "repository_version": "/pulp/api/v3/repositories/container/container/60747422-30d8-4b92-83e6-e6f025b6d829/versions/1/"
+        }
+
+The ``base_path`` differs from the original upstream repo name and configuration should happen per
+repo. Since the original singed itentity differs from the location the images are being served, the
+``remapIdentity`` and full registry path prefix needs to be specified.
```

Comment:
In case you want to publish the signed container images with a different base_path, ...

Reply:
I don't think i understand where the suggestion should go?
The remapidentity should be specified anyway, regardless on whether the base_path is same or different and it would look like this https://github.com/pulp/pulp_container/issues/530#issuecomment-1011015667 The per-repo configuration is needed the moment registry also contains pushed content.
And since mirrored and pushed content is one the base registry usages I decided not to go into the above mentioned semantics to not create more confusion. wdyt?

Reply:
I don't quite remember. I think, i wanted to suggest a more tutorial style (To accomplish that, you need to do...), because i didn't understand why you describe the difference here.

Reply:
I actually had hard time to explain exactly this paragraph. I wanted to outline that because of different content provenance the config should be per repo and that remapidentity should contain full path.
I can probably skip the first sentence and just keep `Since the original singed identity differs from the location the images are being served, the remapIdentity and full registry path prefix needs to be specified.` wdyt?

Reply:
That's spot on. There's a reason, the problem and the action to solve it.

Reply:
pushed